### DISCUSSION TITLE
How to specify Alt + Tab or Enter ?

### DISCUSSION THREAD

**Discussion by Iduoad** [zellij-org/zellij#2477] [Q&A]
I want to Alt+Enter in the config but I get a `Invalid key: 'Alt Tab' ` when I tried this in the config:

```kdl
keybinds {
    shared_except "locked" {
        bind "Alt Tab" { NewTab; SwitchToMode "Normal"; }
        bind "Alt Enter" { NewPane; SwitchToMode "Normal"; }
    }
}
```

**Comment by Lockszmith-GH** [comment]
I'm guessing here that Alt Tab is preserved by the window manager, so can't be bound.

In what environment are you configuring this? (OS, terminal emulator)

**Reply by Iduoad** [reply]
I am using linux with bspwm. Alt Tab is not preserved by the window manager. Also I think it is something more related to Zellij since it give the error on startup.
```
5 │         bind "Alt Enter" { NewPane; SwitchToMode "Normal"; }
   ·         ──────────────────────────┬─────────────────────────
   ·                                   ╰── Invalid key: 'Alt Enter'
```

**Reply by Lockszmith-GH** [reply]
I've answered in the issue ( #2478 ) you opened
